cypress/integration/search_spec.js

```diff
@@ -4,15 +4,14 @@ describe('Search courses fuctionality', () => {
       cy.visit('http://localhost:3000')
 
       cy.get('.search-btn').click()
+        .get('.course-card').first().click()
   })
 
   it('Should filter and display specified courses', () => {
       cy.visit('http://localhost:3000')
 
       cy.get('.search-bar').type('west')
         .get('.search-btn').click()
-
-  // cy.get('.course-card')
-    // .contains('west')
+      cy.get('.card-name').first().contains('Westminster')
```

Comment:
Haven't seen a .first() before if you have time later on would love to know more about it and how its used